build-with-pip.file

```diff
@@ -60,8 +61,10 @@
 %define pip pip3&package_dependency=py3-pip
 Requires: python3 py3-pip py3-setuptools
 %if "%{n}" != "py3-wheel"
+%if "%{?PipDropWheelDep:set}" != "set"
```

Comment:
`PipNoWheelDep`?

Reply:
either `PipNoWheelDep` and then check for `== "set"` or `PipDropWheelDep` and check for `!= "set"` are same :-)

Reply:
I really have no preference here. feel free to change it :-)